src/components/operations/operation-details/ko/runtime/type-definition-combination.html

```diff
@@ -0,0 +1,37 @@
+<!-- ko if: properties && properties.length > 0 -->
+
+<h5 class="combination-type"><span data-bind="text: type.combinationType"></span>:</h5>
```

Comment:
This style assigns margins, but at the same time users have control over H1-H6 sizes, so the will override this one. I'd rather wrapped each such section into DIV and assigned the margin to it.
Another thing, we're collecting that kind of utility classes in utils.scss file. Having it all in one place will help us identify patterns and adjust as we go.